src/extension.ts

```diff
@@ -573,15 +574,16 @@ async function installUpdates (updateInfo: UpdateChoice[] | UpdateInfo[], progre
 			}
 			if (error.metadata) {
 				const { metadata } = error;
-				if (update.productName === updates.ProductNames.AppcInstaller && metadata.errorCode === 'EACCES') {
+				if (update.productName === updates.ProductNames.AppcInstaller || updates.ProductNames.Node && metadata.errorCode === 'EACCES') {
```

Comment:
I think this should be
```suggestion
				if ((update.productName === updates.ProductNames.AppcInstaller || update.productName === updates.ProductNames.Node) && metadata.errorCode === 'EACCES') {
```